idoit-install

```diff
@@ -1147,6 +1147,10 @@ EOF
                 abort "Unable to give read permissions recursively"
             chcon -t httpd_sys_rw_content_t "${INSTALL_DIR}/" -R || \
                 abort "Unable to give write permissions recursively"
+            
```

Comment:
Hi @sirbaas
thank you for the PR. 
I have one request regarding the PR. Could you please wrap your change in an if-statement that checks if SELinux is active at all?

Reply:
Added if statement, what do you think?